eva-accession-import/src/main/java/uk/ac/ebi/eva/accession/dbsnp/persistence/DbsnpVariant.java

```diff
@@ -0,0 +1,33 @@
+package uk.ac.ebi.eva.accession.dbsnp.persistence;
+
+import uk.ac.ebi.eva.commons.core.models.AbstractVariant;
+import uk.ac.ebi.eva.commons.core.models.IVariantSourceEntry;
+import uk.ac.ebi.eva.commons.core.models.VariantStatistics;
+import uk.ac.ebi.eva.commons.core.models.pipeline.VariantSourceEntry;
+
+import java.util.Collection;
+import java.util.Collections;
+import java.util.List;
+
+public class DbsnpVariant extends AbstractVariant {
```

Comment:
I don't think you need this class, `eva-accession` already contains types for submitted and clustered variant, and another one to group them together.